src/main/java/uk/gov/hmcts/cmc/claimstore/services/InterestRateService.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.domain.models.InterestAmount;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+@Component
+public class InterestRateService {
```

Comment:
I guess this service should be used in all places where interest calculation is needed so we have one component responsible for calculation. Have you tried to find other places where we calculate interest and reuse this service in such places?

Reply:
I wanted to keep this PR as small as possible. Only new endpoint and tests for that.

Then next one - using it everywhere, where it should be used.

And one more for FE to call API whenever it should be calculated.

Reply:
I just looked into the code and found `uk.gov.hmcts.cmc.claimstore.services.interest.InterestCalculationService#calculateInterest ` method which seems to do exactly what this service is going. Please reuse existing one or merge these two.